styles/styles.css

```
@@ -61,65 +103,43 @@ li {
 .navigation-choice {
     font-family: inherit;
```

```suggestion
font-size: поставь по макету!
```

Ссылки наследуют font-family. Его не нужно указывать ещё раз.